components/AppFooter/AppFooter.tsx

```diff
@@ -121,7 +121,10 @@ export const AppFooter = () => {
           </Breadcrumbs>
         )}
         <p className={classes.copyright}>
-          &copy;{copyrightDate} The World from PRX
+          &copy;{copyrightDate} The World from PRX.
```

Comment:
Why add a period? This is not a sentence.

Reply:
@brandonhundt ^^

Reply:
@rpeterman-gp I just addressed this! Good catch.